### PR TITLE
Add script to fetch github users-repo mapping

### PR DESCRIPTION
**Pull request by harishv7**
Adding script to fetch list of repositories and associated users of that repo using Octokit

### CODE REVIEW

**Review by kishore03109 — COMMENTED** [review]
How accurate do we need this script to be ah? I think we can improve this here quickly by hardcoding the list of known repos that are not related to the sites. eg. `isomercms-frontend`, `redirection`, etc.

**Comment by harishv7** [comment]
@kishore03109 added repo check

File: get-github-users-repo-mapping/index.js [hunk 1]
```diff
@@ -0,0 +1,95 @@
...
+      userCsvStream.write({ Repo: repo.name, UserID: collaborator.login });
+    }
+  }
+
+  repoCsvStream.end();
+  userCsvStream.end();
+}
+
+const orgName = "isomerpages";
+getUsers(orgName);
```
Comment (QiluXie) [inline get-github-users-repo-mapping/index.js:95]: nit and non-blocking: 
is it better to also handle the promise rejection for this top level async call, print out the error, and exit with a proper error code?  
Something like:
```js
(async function() {
    try {
        await getUsers(orgName);
    } catch (err) {
        console.log(err); 
        process.exit(1);
    }
})();
```

**Review by QiluXie — APPROVED** [review]
left one non-blocking comment, approved

**Pull request merged**